smart-contracts/contracts/mixins/MixinKeys.sol

```diff
@@ -228,6 +243,24 @@ contract MixinKeys is
     return _ownerOf[_tokenId];
   }
 
+  /**
+  * @notice Update transfer and cancel rights for a given key
+  * @param _tokenId The id of the key to assign rights for
+  * @param _keyManager The address to assign the rights to for the given key
+  * setting to address(0) disables transfers for this key until it expires
+  * */
+  //
+  function setKeyManagerOf(
+    uint _tokenId,
+    address _keyManager
+  ) public
+    isKey(_tokenId)
+  {
+    require((msg.sender == keyManagerOf[_tokenId]) || (isLockManager(msg.sender) == true), 'SETKEYMANAGEROF_ACCESS_DENIED');
```

Comment:
For consistent syntax / style don't include `== true` when checking a bool. The params here are also not needed.

Reply:
I will remove the `== true`. thx for the consistency reminder. 
not sure about the params though... `isLockManager` takes an address and `keyManagerOf` needs a uint. I know I must be missing something very obvious.

Reply:
This is minor of course, but I mean this is the same `msg.sender == keyManagerOf[_tokenId] || isLockManager(msg.sender)`